apps/docs/src/app/core/component-docs/card/examples/card-example.component.scss

```diff
@@ -0,0 +1,7 @@
+.fd-docs-card-fix-width {
+    width: 18.75rem;
+}
+
+.fd-docs-card-content-image {
+    margin-bottom: -0.2rem;
```

Comment:
so if user wants to have an image they have to add this margin-bottom?

Reply:
image takes extra space at bottom.
have removed it now. as overriding is not required.